### ISSUE TITLE
Improving the Anime Section

### ISSUE THREAD

**Issue by suraj-9849** [Sulagna-Dutta-Roy/GGExtensions#580]
![image](https://github.com/Sulagna-Dutta-Roy/GGExtensions/assets/146669517/40cee339-bccb-4e97-a12c-25f3bf5dc8fe)
I would like to improve this section by adding the another api from the https://docs.consumet.org/ this will also give the report of the animes and can help the people to choose which anime they would like to see this will help the user to make the decesions. so I would like to work on this issue please assign this to me under @gssoc @Sulagna-Dutta-Roy

**Comment by suraj-9849** [comment]
@Sulagna-Dutta-Roy mam I am implementing the Backend for it. is it ok like I will get the data from the Gogoanime and then show the data on search like any anime can be shown @Sulagna-Dutta-Roy

**Comment by suraj-9849** [comment]
![image](https://github.com/Sulagna-Dutta-Roy/GGExtensions/assets/146669517/0aa9e483-a493-421d-beff-79b3ce8da062)
![image](https://github.com/Sulagna-Dutta-Roy/GGExtensions/assets/146669517/dc8f385e-b452-4cda-b3d2-f5e025a303ff)
![image](https://github.com/Sulagna-Dutta-Roy/GGExtensions/assets/146669517/103e98cd-f75c-48f3-8c13-10547589504e)
I will make it responsive and also add the best CSS too is it ok mam @Sulagna-Dutta-Roy

**Comment by suraj-9849** [comment]
![image](https://github.com/Sulagna-Dutta-Roy/GGExtensions/assets/146669517/aaaad1ae-d0ba-4b68-9804-9fcd0615fd3b)

**Comment by suraj-9849** [comment]
https://github.com/Sulagna-Dutta-Roy/GGExtensions/assets/146669517/227eaccb-4a2b-4545-8e79-0f460d44413f

There You go Mam @Sulagna-Dutta-Roy if this is ok then I will make the PR by adding the Quotes 
This search was implemented to the user for searching the Anime with the necessary information like When it is released and Sub or Dub and The main he can watch by clicking on the Watch now button and also added the Hover effect on the anime-items this will provide the best experience for the user.

**Comment by suraj-9849** [comment]
Reply mam please @Sulagna-Dutta-Roy

**Comment by Sulagna-Dutta-Roy** [comment]
Can you work with JavaScript? Because already in the project description mentioned about the tech stacks like: Html Css JavaScript only.

**Comment by suraj-9849** [comment]
![image](https://github.com/Sulagna-Dutta-Roy/GGExtensions/assets/146669517/2fa094d0-921f-4082-b0e3-c7bbd4cf37c7)
 I am Sorry mam but the API which I am using is in the type Module format this cannot be Done. But It is Not that Backend which I implemented. If there is a problem then I can even write the readme to this . This can help any one It is user friendly code mam U can take it. @Sulagna-Dutta-Roy

**Comment by suraj-9849** [comment]
But still I will make a Try @Sulagna-Dutta-Roy

**Comment by suraj-9849** [comment]
Yeah Mam this can Be possible give me some time to do it. Please increase the level mam @Sulagna-Dutta-Roy

**Comment by suraj-9849** [comment]
https://github.com/Sulagna-Dutta-Roy/GGExtensions/assets/146669517/1e9ba2f3-1a3a-4d24-a6d6-bde7b95c6de0

This is done using the jikan API without backend and I use the html,css & js this doesnot even need any API key which can help the user to directly get the results without adding his own API key by going to the code . If this is ok then I will make the PR mam @Sulagna-Dutta-Roy

**Comment by suraj-9849** [comment]
Reply mam please @Sulagna-Dutta-Roy

**Comment by Sulagna-Dutta-Roy** [comment]
Sure. Work on this. Will change the label after creating a PR.
Also my repository is only for html,css,javascript only for beginners.

**Comment by Sulagna-Dutta-Roy** [comment]
Try not to add any backend dependencies because not supporting that. Otherwise close this Issue

**Comment by suraj-9849** [comment]
Ok mam . There is no backend dependencies 
I am making the PR

**Comment by Sulagna-Dutta-Roy** [comment]
Received the PR. Closing this Issue
